.github/workflows/deploy-all.yml

```diff
@@ -111,6 +111,11 @@ jobs:
         working-directory: "./devops/"
         run: |
           make oc-deploy-web
+      # Deploy Forms
```

Comment:
I do not expect the Form.io server to be deployed every time in the same way that I do not expect Redis or Patroni.
Maybe I am missing something here, what is the idea?

Reply:
its just the deploying the forms from the latest build. We dont want to do it everytime, but as the name states all deploy, it was included.

Reply:
I believe it is the deployment of form definitions, isn't it?